static/js/tools/timeline/__snapshots__/page.test.tsx.snap

```diff
@@ -1,169 +1,169 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Single place and single stat var 1`] = `
-"<div class=\\"chart-container dc-async-element-holder\\">
-  <div class=\\"card\\">
-    <div class=\\"statVarChipRegion\\">
-      <div class=\\"chip\\"><span class=\\"chip-text chip-clickable-text\\">Age</span><button class=\\"chip-action\\"><i class=\\"material-icons\\">cancel</i></button></div>
+"<div class="chart-container dc-async-element-holder">
```

Comment:
is this updated because of jest updates?

Reply:
Yes!